src/Client.js

```diff
@@ -1,35 +1,36 @@
 import Particle from './Particle';
 import Library from './Library';
 
+
 export default class Client {
-	constructor({ auth, api = new Particle() }) {
+	constructor({ auth, api = new Particle() }){
 		Object.assign(this, { auth, api });
 	}
 
-	ready() {
+	ready(){
 		return Boolean(this.auth);
 	}
 
 	/**
 	 * Get firmware library objects
 	 * @param  {Object} query The query parameters for libraries. See Particle.listLibraries
-	 * @return {Promise}
+	 * @returns {Promise} A promise
```

Comment:
a bit silly since we should be specifying exactly what is the promise is resolved & rejected with but it's enough to placate the linter and not any worse than what we were doing